stable/sonatype-nexus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.12.1
+version: 1.12.2
```

Comment:
you should bump the minor version since the chart breaks the ux compatibility, i.e. `nexus.serviceType` -> `nexus.service.type`.

Reply:
fixed